test/e2e/flask/multichain-api/wallet_invokeMethod.spec.ts

```diff
@@ -71,17 +73,10 @@ describe('Multichain API', function () {
                 `[data-testid="invoke-method-${scope}-btn"]`,
               );
 
-              const resultElement = await driver.findElement(
-                `#invoke-method-${escapeColon(scope)}-${invokeMethod}-result-0`,
-              );
-
-              const result = await resultElement.getText();
-
-              assert.strictEqual(
-                result,
-                `"${EXPECTED_RESULTS[scope]}"`,
-                `${scope} method ${invokeMethod} expected "${EXPECTED_RESULTS[scope]}", got ${result} instead`,
-              );
```

Comment:
this is an anti-pattern that leads to race conditions. We should look for the specific text
See more here https://github.com/MetaMask/metamask-extension/issues/19870

Reply:
ahhh. yeah that makes sense